test/method/testnuclear.py

```diff
@@ -74,7 +74,7 @@ def test_repulsion_energy(self) -> None:
             output = f.read()
         line = re.search('Nuclear Repulsion Energy = .* hartrees', output).group()
         nre = float(line.split()[4])
-        nre = utils.convertor(nre, 'Angstrom', 'bohr')
+        nre = utils.convertor(nre, 'eV', 'hartree')
```

Comment:
Remember that this is the value coming directly from the output file, which will always be in atomic units, so
```suggestion
        nre = utils.convertor(nre, 'hartree', 'eV')
```